src/components/Table/Table.jsx

```diff
@@ -41,9 +41,7 @@ class Table extends React.Component<Props, State> {
     empty: false,
     emptyText: '',
     loading: false,
-    stickAt: null,
     headerData: [],
-    sort: {},
```

Comment:
are these default values not used?

Reply:
it was always protected by `sort && ...`, and it caused flow issues with the tests (where it was annotated as a required attribute)